src/travelers/Traveler.js

```diff
@@ -40,6 +45,19 @@ class Traveler {
 
     return sortedTrips;
   }
+
+  getEstimatedCostForTrip(listOfDestinations, userInputs) {
+    const destination = new Destination(listOfDestinations.find(d => d.id === userInputs.destinationID))
+    console.log(destination)
```

Comment:
Don't forget to remove the couple console logs in this file

Reply:
Will do!